src/napari_cryoet_data_portal/_reader.py

```diff
@@ -211,10 +266,74 @@ def read_annotation(annotation: Annotation, *, tomogram: Optional[Tomogram] = No
     else:
         attributes["name"] = f"{tomogram.name}-{name}"
     attributes["metadata"] = annotation.to_dict()
-    attributes["face_color"] = OBJECT_COLOR.get(name.lower(), DEFAULT_OBJECT_COLOR)
+    attributes["face_color"] = OBJECT_COLOR.get(name, DEFAULT_OBJECT_COLOR)
+    return data, attributes, layer_type
+
+
+def read_annotation_files(annotation: Annotation, *, tomogram: Optional[Tomogram] = None) -> Generator[FullLayerData, None, None]:
+    """Reads multiple annotation layers.
+
+    Parameters
+    ----------
+    annotation : Annotation
+        The tomogram annotation.
+    tomogram : Tomogram, optional
+        The associated tomogram, which may be used for other metadata.
+
+    Yields
+    -------
+    napari layer data tuple
+        The data, attributes, and type name of the layer that would be
+        returned by `Points.as_layer_data_tuple` or `Labels.as_layer_data_tuple`.
+
+    Examples
+    --------
+    >>> client = Client()
+    >>> annotation = client.find_one(Annotation)
+    >>> for data, attrs, typ in read_annotation_files(annotation):
+            layer = Layer.create(data, attrs, typ)
+    """
+    for f in annotation.files:
+        if (f.shape_type in ("Point", "OrientedPoint")) and (f.format == "ndjson"):
+            yield _read_points_annotation_file(f, anno=annotation, tomogram=tomogram)
+        elif (f.shape_type == "SegmentationMask") and (f.format == "zarr"):
+                yield _read_labels_annotation_file(f, anno=annotation, tomogram=tomogram)
```

Comment:
weird indenting?

Reply:
Weird indeed. Thanks for the catch!

Reply:
Ah, I have ruff configured for pre-commit, but did not install it locally or have CI to fix stuff. Will follow up with a separate PR to fix all the formatting.